.github/workflows/release-push-image.yaml

```diff
@@ -118,10 +116,24 @@ jobs:
       - name: Install Helm
         uses: azure/setup-helm@fe7b79cd5ee1e45176fcad797de68ecaf3ca4814 # v4.2.0
 
-      - name: Publish helm chart
-        uses: step-security/helm-gh-pages@6a390e89293c1ec8bc5120f6692f3b8a313a9a3d # v1.7.0
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@0d4c9c5ea7693da7b068278f7b52bda2a190a446 # v3.2.0
         with:
-          target_dir: charts
-          token: ${{ secrets.GITHUB_TOKEN }}
-          branch: gh-pages
-          app_version: ${{ env.VERSION }}
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract version
+        id: extract_version
+        run: |
+          VERSION=$(grep 'version=' gradle.properties | cut -d '=' -f2)
+          echo "VERSION=${VERSION}" >> $GITHUB_ENV
```

Comment:
We need to get the version from the file, since the version of the helm to publish is in lockstep with the version of the tagged release and the version of the block-node release.

Reply:
Just because they do this doesn't mean it's ideal; both of those might just be a legacy or gradle-ism.
Why should the helm chart version _necessarily_ be identical to the tagged release and block-node release?  I would think a helm chart can have different release and update cycles than the software it deploys without breaking...

Reply:
I agree with you that the helm version and the tagged release version of the block-node don't need to be in lockstep, however is the convention we've been using and currently is the same workflow for both artifacts... for simplicity sake and since we are using the same repo, I would argue against dividing the workflows and separating the versions unless there is a clear need that compels us to do it.

Reply:
That's reasonable.  This will work fine for now; though we may want to revisit in the future, as we further mature the product and processes.